### DISCUSSION TITLE
Add function `etna.datasets.load_dataset` to public API

### DISCUSSION THREAD

**Discussion by ArtemLiA** [etna-team/etna#447] [Improvements]
### Description

Hi everyone! I noticed that the `etna.datasets.load_dataset` function is not currently in the API References, and its only mention is on the "Internal datasets" page on the "User guide" section. I think we should fix this and add `load_dataset` function to public API.

### Thoughts

It seems that this can be easily implemented by adding the import of the `load_dataset` function in the file `__init__.py`.

**Comment by d-a-bunin** [comment]
Hi! I agree. It is probably a good idea to also cross-reference pages:
- Make in a "User guide" page link to `load_dataset`, it will allow users to check the parameters
- Make in `load_dataset` reference to "User guide" page for users to see the list of all available datasets.

May be we should also add `etna.datasets.internal_datasets.list_datasets` more public.

@ostreech1997, what do you think?

**Comment by d-a-bunin** [comment]
Solved in #465.
